magenta_docker/user_interaction.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+import os
+
+# Determining path for data retrieval
+data_path = raw_input('Insert a path for data retrieving. '
+'This must be a GCS Bucket:\n')
```

Comment:
Here and below: This line should come under `'Insert`. Have you processed this file agains py_lint?

Reply:
No, I will do it right now and make the changes.